cmd/manager/main.go

```diff
@@ -207,6 +207,8 @@ func init() {
 		"Select a namespace for the controller loop. If not set, uses ${POD_NAMESPACE} environment variable")
 	rootCmd.PersistentFlags().DurationVar(&flags.controllerOptions.ResyncPeriod, "resync", time.Second,
 		"Controller resync interval")
+	rootCmd.PersistentFlags().StringVar(&flags.controllerOptions.DomainSuffix, "domainSuffix", "cluster.local",
```

Comment:
Localkube calls it `dns-domain`, kubelet calls it `cluster-domain`. Seems like a lot of options :)